### PR TITLE
Wallet implement multiple delegation

### PR DESCRIPTION
**Pull request by jpcapurro-atixlabs**

### CODE REVIEW

**Comment by jpcapurro-atixlabs** [comment]
@dmernies-iohk could you review this?

File: examples/wallet/app/components/StakePoolList.js [hunk 1]
```diff
@@ -1,41 +1,72 @@
...
+                {newDelegationPercentage}%
+              </Col>
+              <Col xs={2}>{currentDelegationPercentage}%</Col>
+              <Col xs={3}>
+                <Row className="justify-content-between">
+                  <Button
+                    type="button"
+                    disabled={!newDelegationPercentage}
+                    size="sm"
+                    onClick={() => onSelection(poolId, -1)}
```
Comment (enzoc4) [inline examples/wallet/app/components/StakePoolList.js]: I'd like to give this `-1` some name like `increment`, but I don't see an easy way

File: examples/wallet/app/components/StakePoolList.js [hunk 2]
```diff
@@ -1,41 +1,72 @@
...
+            currentDelegation && percentageFromParts(currentDelegation, poolId);
+          return (
+            <ListingRow key={poolId}>
+              <Col style={rowStyles} className={styles.poolId} xs={4}>
+                {poolId}
+              </Col>
+              <Col style={rowStyles} xs={2}>
+                {newDelegationPercentage}%
+              </Col>
+              <Col xs={2}>{currentDelegationPercentage}%</Col>
```
Comment (enzoc4) [inline examples/wallet/app/components/StakePoolList.js]: Just a comment, but it took me some time initially to understand what this column was, although it is just a minor thing

File: examples/wallet/app/components/StakeDelegation.js [hunk 3]
```diff
@@ -1,38 +1,134 @@
...
+  const [delegation, setDelegation] = useState<NewDelegation>({});
+  const [selectionCount, setSelectionCount] = useState<number>(0);
+
+  const poolSelectionHandler: PoolSelectionHandler = (poolId, increment) => {
+    const currentDelegation =
+      (delegation[poolId] && delegation[poolId].parts) || 0;
+    setDelegation(
+      Object.assign({}, delegation, {
+        [poolId]: Object.assign(
+          { color: colors[selectionCount % colors.length], parts: 0 },
```
[the remark this answers is not in the document]
Comment (jpcapurro-atixlabs) [reply examples/wallet/app/components/StakeDelegation.js]: correct, flow assumes `Object.assign` returns the type of the first argument.

File: examples/wallet/app/components/StakePoolList.js (hunk 2 above)
Comment (jpcapurro-atixlabs) [reply examples/wallet/app/components/StakePoolList.js]: I think it'll be more clear when the table has headers :upside_down_face:

**Pull request merged**